rviz_visual_testing_framework/CMakeLists.txt

```diff
@@ -109,6 +109,12 @@ install(
   DESTINATION include
 )
 
+install(FILES
```

Comment:
I believe it's mostly aesthetic, but I like how in #379 the rviz config files were moved to a `config` directory that is then installed.